request.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+ /* Call The api and return the response the JSON file.
+ *
+ * @param string $url
+ *   Url is taking as input to call api.
```

Comment:
Give one empty line after param description and return variable description

Reply:
Done